### PR TITLE
feat wasm: expose getAllocationMinLock, getRemoteFileMap 

### LINKED ISSUE #793: feat wasm expose: GetRemoteFileMap
https://0chain.slack.com/archives/G014PQ61WNT/p1677010900218749?thread_ts=1677005759.095129&cid=G014PQ61WNT

### PR DESCRIPTION
**Pull request by boddumanohar**
### Changes
-

### Fixes
- https://github.com/0chain/gosdk/issues/793

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by peterlimg — APPROVED** [review]
LGTM

File: wasmsdk/allocation.go [hunk 1]
```diff
@@ -134,3 +137,66 @@ func updateAllocation(allocationID string,
...
+				EncryptedKey: data.EncryptedKey,
+				LookupHash:   data.LookupHash,
+				CreatedAt:    data.CreatedAt,
+				UpdatedAt:    data.UpdatedAt,
+			},
+		}
+		fileResps = append(fileResps, resp)
+	}
+
+	return getJSON(fileResps)
```
Comment (cnlangzi) [inline wasmsdk/allocation.go]: it is better to return `fileResps` directly instead of getJSON(fileResps).

**Review by cnlangzi — DISMISSED** [review]
a small fix

File: wasmsdk/allocation.go (hunk 1 above)
Comment (ranajahanzaib) [reply wasmsdk/allocation.go]: @boddumanohar, I agree with him.

Comment (boddumanohar) [reply wasmsdk/allocation.go]: updated.

**Pull request merged**